simulator/src/resim/cmd_publish.rs

```diff
@@ -140,7 +141,7 @@ impl Publish {
             let mut vm_type_updates = index_map_new();
             let mut original_code_updates = index_map_new();
             let mut instrumented_code_updates = index_map_new();
-            let instrumented_code = ScryptoV1WasmValidator::new(0u64)
+            let instrumented_code = ScryptoV1WasmValidator::new(ScryptoVmVersion::V1_0)
```

Comment:
Not sure here. Should we use `V1_0` or latest?

Reply:
Ok, found scenario where it fails for `V1_0`.
- Prepare a new package that includes features added in `V1_1` or later.
- Attempt to update already published package with the new one

Will change to `ScryptoVmVersion::latest()`